Wordle.Api/Controllers/WeatherForecastController.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.AspNetCore.Mvc;
```

Comment:
I doubt it will be an issue when grading, but the API files were pushed.  You can probably leave them in, I just wanted to call it out if these found their way into the wrong local repository.